From b98c28c4883de6885a3ef8c8f0db757d50cb26b5 Mon Sep 17 00:00:00 2001
From: Bingwu Zhang <xtexchooser@duck.com>
Date: Mon, 19 Aug 2024 19:32:04 +0800
Subject: [PATCH 1/2] Fix versioning
X-Developer-Signature: v=1; a=openpgp-sha256; l=623; i=xtexchooser@duck.com;
 h=from:subject; bh=5qQJ1ep2+26xdHrpffyWlmCKZjlrKhfAJqTI0FX2EOw=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGmH3eazTgqKMbOdOrlG8r74tSMuV75xy0UnvLSzPuad5
 WfTdsK/o5SFQYyLQVZMkaXIsMGbVSedX3RZuSzMHFYmkCEMXJwCMJEn8owM874tLGHIDJb+ttma
 8bPfUg3rH3yLZJYs06oVC9TRZTn4hOGfumiO02yGX8mSci83itWrT15bPbk/9qNSvYze+bW2om8
 YAQ==
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

Link: https://github.com/AcademySoftwareFoundation/Imath/issues/409
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9770a90e4f62..d8d548d7d1f1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -14,7 +14,7 @@ endif()
 
 # Imath version
 
-project(Imath VERSION 3.1.10 LANGUAGES C CXX)
+project(Imath VERSION 3.1.11 LANGUAGES C CXX)
 
 set(IMATH_VERSION_RELEASE_TYPE "" CACHE STRING "Extra version tag string for Imath build, such as -dev, -beta1, etc.")
 

base-commit: 5ac1d5335cb34f0f356c5f2461a57c845b96b115
-- 
2.46.0

